tests/integration/server/roles_api_test.py

```diff
@@ -48,3 +71,27 @@ def test_create_duplicate_role():
         assert response.json()["code"] == 400
         delete_user_by_username("susan22")
         delete_role_by_name("Manager1")
+
+
+def test_syncdirectionflag_rolename():
+    """ Testing the presence and the value of syncdirection flag  is set to OUTBOUND of a role in roles table. """
```

Comment:
Too many characters in doc strings, please make your doc strings two or three lines versus one long line.

Reply:
But both the Linter & Formatter successfully 100% passed.

Reply:
Linter and formatter doesn't look at doc strings, I forgot why docstrings weren't formatted, but according to PEP8 standards, docstrings should be no longer than 72 characters:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
I get it, but all these rules should be incorporated into our LINTER & FORMATER codes. Also, in our existing code base there are several examples where you can see the doc strings length more than 85.

![image](https://user-images.githubusercontent.com/23491817/55592894-c9d69800-56ee-11e9-9786-9c2a65ec201d.png)

Reply:
My apologies - I must have missed that when I was reviewing the change. But yeah - it would be great if we could incorporate it into our linter or formatter. We can possibly create a bug ticket.